src/log/log_slot.c

```diff
@@ -33,7 +33,7 @@ __log_slot_dump(WT_SESSION_IMPL *session)
             earliest = i;
         __wt_errx(session, "Slot %d (0x%p):", i, (void *)slot);
         __wt_errx(session, "    State: %" PRIx64 " Flags: %" PRIx32, (uint64_t)slot->slot_state,
```

Comment:
Minor: The `PRIx32` needs to be changed to `PRIx16`. I'm surprised there was no compiler warning for that.

Reply:
Done.